stock_picking_invoice_link/models/stock_move.py

```diff
@@ -28,6 +28,7 @@ def _compute_invoice_line_id(self):
     def _create_invoice_line_from_vals(self, move, invoice_line_vals):
         inv_line_id = super(StockMove, self)._create_invoice_line_from_vals(
             move, invoice_line_vals)
-        move.invoice_line_ids = [(4, inv_line_id)]
+        move.with_context(skip_update_line_ids=True).invoice_line_ids = [
```

Comment:
@sergiocorato Is this 'skip_update_line_ids') present in core (I could not find it)?

Reply:
@rousseldenis no, it is in `stock-logistick-workflow` repo, in module `stock_picking_package_preparation_line`